gprofiler/profilers/java.py

```diff
@@ -966,11 +1011,22 @@ def _profile_process(self, process: Process, duration: int, spawned: bool) -> Pr
             self._jattach_timeout,
             self._ap_mcache,
         ) as ap_proc:
-            stackcollapse = self._profile_ap_process(ap_proc, comm, duration)
+            stackcollapse, process_uptime_str = self._profile_ap_process(ap_proc, comm, duration)
```

Comment:
Is it possible to do all "collect uptime" logic here, without changing `AsyncProfiledProcess` itself? (which tries to remain about async-profiler and not other stuff). Also will simplify the code if we manage to